autopush/main.py

```diff
@@ -532,6 +533,9 @@ def connection_main(sysargs=None, use_files=True):
     resource = DefaultResource(WebSocketResource(factory))
     resource.putChild("status", StatusResource())
     site_factory = Site(resource)
+    # Silence starting/stopping messages
+    site_factory.noisy = args.debug
+    site.noisy = args.debug
```

Comment:
fair enough. I thought I was being overly assertive with the base class, but 100% OK with being even more assertive.